### PR TITLE
Updated README.es.md, And Translated INSTALL.md, CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ucdstudent95618**
I have updated the README.md with some minor fixes and have translated both INSTALL.md, CONTRIBUTING.md to Spanish :)

### CODE REVIEW

File: README.es.md [hunk 1]
```diff
@@ -17,11 +19,11 @@ Para una documentación más detallada, ver https://pyre-check.org.
 
 ## Instalación
 
-Consulte [INSTALL.md](https://github.com/facebook/pyre-check/blob/master/INSTALL.md) para obtener detalles sobre la instalación de Pyre desde una versión empaquetada o desde el origen.
+Consulte [Instalación](INSTALL.es.md) para obtener detalles sobre la instalación de Pyre desde una versión empaquetada o desde el origen.
 
 ## Únete a la comunidad de Pyre
 
-Consulte [CONTRIBUTING.md](https://github.com/facebook/pyre-check/blob/master/CONTRIBUTING.md) para saber cómo ayudar.
+Consulte [Contribuir](CONTRIBUTING.es.md) para saber cómo ayudar.
```
Comment (sinancepel) [inline README.es.md]: Minor detail: I'd keep these names consistent with the original "[CONTRIBUTING.es.md]" and "[INSTALL.es.md]".

**Comment by sinancepel** [comment]
Wow, this is very detailed - thank you, once again, for contributing. Only one small detail :)

**Comment by ucdstudent95618** [comment]
Fixed the links :)  Let me know if there is anything else I should fix. Thanks!

**Comment by sinancepel** [comment]
🎉

**Pull request closed without merging**